### DISCUSSION TITLE
Getting exit code = 1 even with Overlay Buttons disabled

### DISCUSSION THREAD

**Discussion by Issues66010222** [kitadai31/revanced-patches-android6-7#45] [Q&A]
Maybe I should share my manager logs first:
[revanced-manager_logcat_20230818130024001352.log](https://github.com/kitadai31/revanced-patches-android6-7/files/12375573/revanced-manager_logcat_20230818130024001352.log)

Okay, first of all, my device is nonroot, supports armeabi-v7a and armeabi archs, and has Android 6.0.1 OS.

I don't know what I did wrong, what's going wrong, or what I'm missing; I followed [the instructions](https://github.com/kitadai31/revanced-patches-android6-7/wiki/How-to-build) thoroughly, at least the ones that are applicable to my Samsung Galaxy Grand Prime Plus. It's not just the `exit code = 1` that I'm having trouble with, I've noticed other errors too: 
[revanced-manager_patcher_20230818133233759105.log](https://github.com/kitadai31/revanced-patches-android6-7/files/12375769/revanced-manager_patcher_20230818133233759105.log) I should mention that this is my third attempt, I forgot to export the previous logs where it made it halfway through and said said a bunch of stuff like `'settings' failed previously`

**Comment by kitadai31** [comment]
arm64-v8a is required...
